rest-service/manager_rest/models.py

```diff
@@ -40,6 +40,15 @@ def __init__(self, **kwargs):
         self.updated_at = kwargs['updated_at']
 
 
+class Snapshot(SerializableObject):
+
+    fields = {'id', 'created_at'}
+
+    def __init__(self, **kwargs):
+        self.id = kwargs['id']
```

Comment:
We also want a status here - like CREATING / UNDER_CONSTRUCTION, CREATED, FAILED